src/APBFramelessWindow/APBFramelessWindowTitleBar.h

```diff
@@ -25,7 +25,6 @@ class APBFramelessWindowTitleBar: public QWidget {
 
 	private:
 	QLabel *windowTitle;
-
```

Comment:
Clean code principle: `Use white space to associate related things and disassociate weakly related.`